paack-php-sdk/src/Tracking/Test/TestTracking.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+use Paack\Config\Mixes;
+use Paack\Init;
+use Paack\Label\Logic\LabelSchema;
+use Paack\Order\Logic\AddressSchema;
+use Paack\Order\Logic\CustomerSchema;
+use Paack\Order\Logic\ExtraDetail;
+use Paack\Order\Logic\MoneySchema;
+use Paack\Order\Logic\OrderSchema;
+use Paack\Order\Logic\ParcelSchema;
+use Paack\Order\Logic\TimeSlot;
+use PHPUnit\Framework\TestCase;
+use Paack\Label\Interfaces\LabelFormat;
+
+class TestTracking extends TestCase {
+
+    protected array $obj_array = [];
+    private Init $init;
+
+    public function setUp(): void
+    {
+        //$this->init = new Init( getenv('CLIENT_ID'), getenv('CLIENT_SECRET'), 'staging');
+        $this->init = new Init( '47SoYRqAZWdd26Cify0fwLlxtsO50F4R', 'MxVunSMrU-8UDOqpUeR7aTzVF7lB4XVz-8jjEjsO4tN_4xZlg54_iTGlnQYN9Plp', 'staging');
```

Comment:
@paack-andreidraghia secrets need to be removed

Reply:
Done, from all places. Sorry for missing this.